backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/utils/ApplicationHandler.kt

```diff
@@ -62,6 +62,30 @@ class ApplicationHandler(
         Mailer.sendNotificationForApplicationMails(project, backendConfig, projectConfig, regionId)
     }
 
+    fun sendPreVerifiedApplicationMails(
+        applicationEntity: ApplicationEntity,
+        verificationEntities: List<ApplicationVerificationEntity>,
+        dataFetcherResultBuilder: DataFetcherResult.Builder<Boolean>
+    ) {
+        val backendConfig = context.backendConfiguration
+        val projectConfig = backendConfig.projects.first { it.id == project }
+
+        for (applicationVerification in verificationEntities) {
```

Comment:
even we should only have one organization in this case for verein360, in theory there could be more organization that would all receive this e-mail.